packages/yourloops/components/statistics/standard-deviation-stat.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+import React, { type FunctionComponent } from 'react'
+import { type BgPrefs, CBGStandardDeviation } from 'dumb'
+import Box from '@mui/material/Box'
+import { t } from 'i18next'
+import { type BgType } from 'medical-domain'
+
+export interface StandardDeviationProps {
+  bgPref: BgPrefs
+  bgType: BgType
+  averageGlucose: number
+  standardDeviation: number
+  standardDeviationTotal: number
+}
+
+export const StandardDeviationStat: FunctionComponent<StandardDeviationProps> = (props) => {
+  const { standardDeviation, averageGlucose, standardDeviationTotal, bgPref, bgType } = props
+  const defaultAnnotations = [t('standard-deviation-tooltip'), t('tooltip-empty-stat')]
+  const standardAnnotation = t('standard-deviation-tooltip')
+
+  const formatData = (data: number): number => {
+    return Math.round(data)
+  }
+
+  const getAnnotations = (selectedLabel: string): string[] => {
+    const hasValidValue = standardDeviation | averageGlucose
+    switch (selectedLabel) {
+      case 'cbg':
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏It's not a very good practice to use raw strings in the code (they're called "magic strings") because it makes the code harder to maintain. Here you should reuse the type directly: `DatumType.Cbg` and `DatumType.Smbg`